lang/zh-cn.json

```diff
@@ -292,6 +292,8 @@
         "OneVisit": "1 次访问",
         "OnlyEnterIfRequired": "在您的 SMTP 服务器需要用户名时才需要输入",
         "OnlyEnterIfRequiredPassword": "在您的 SMTP 服务器需要密码时才需要输入",
+        "UsuallyYourEmailAddress": "通常是您的Email地址",
```

Comment:
Unfortunately, we can only include english default translations. Other languages will need to be translated through our translation tool see https://matomo.org/translations